client_plugin/drivers/shared/shared_driver.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
```

Comment:
Hmmm I know that the golang naming convention doesn't recommend `_` in package name. But I didn't know that it's also not recommended in filename.
Can you confirm?
I saw `_` in filenames almost everywhere in the codebase :)